scripts/update-issues.ts

```diff
@@ -0,0 +1,233 @@
+import { writeFile } from "node:fs/promises";
+import { features } from "web-features";
+import bcd from "@mdn/browser-compat-data" with { type: 'json' };
+
+import { Octokit } from "@octokit/rest";
+import { throttling } from "@octokit/plugin-throttling";
+
+const dryRun = process.argv.includes("--dry-run");
+
+// A special comment in the issue body is used to store the web-features
+// ID, <!-- web-features:some-feature -->. Whitespace is allowed wherever
+// possible to make the matching less brittle to changes.
+const pattern = /<!--\s*web-features\s*:\s*([a-z0-9-]+)\s*-->/;
+
+// A mapping of Mozilla and WebKit standards positions for features are
+// maintained for the web-features explorer. Use that data to skip features that
+// have a negative/oppose position, as suggested by Mozilla.
+const postitionsUrl =
```

Comment:
Can we add a note about the fact that this data is moving to https://github.com/web-platform-dx/web-features-mappings/ soon.
I don't have a lot of time to work on this over the coming weeks, but as soon as we have something as complete as what's on the explorer, and stable, I'll work to remove the data from the explorer, and pull it from https://github.com/web-platform-dx/web-features-mappings/ instead.